modules/sor/sorV2/sorPathService.ts

```diff
@@ -505,7 +506,11 @@ class SorPathService {
             include: prismaPoolAndHookWithDynamic.include,
         });
 
-        const allPools = [...pools, ...lbps];
+        // always include MEV_CAPTURE hooks, even if considerPoolsWithHooks is false and we dont want to include hooks
+        const allPools = [
+            ...pools.filter((pool) => considerPoolsWithHooks || (pool.hook as HookData).type === 'MEV_CAPTURE'),
```

Comment:
How does this filter work? `considerPoolWithHooks` is simply a boolean flag 🤔

Reply:
Feel free to change it to whatever way makes most sense to you :)